DataFormats/L1TrackTrigger/interface/TTDTC.h

```diff
@@ -17,7 +17,7 @@
 class TTDTC {
 public:
   // bit accurate Stub
```

Comment:
Consider moving definition of types BV, Frame, Stream & Streams to TTTypes.h, so algos like HT don't need unexpected references to TTDTC class (unexpected, as HT doesn't read data from DTC).